staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller.go

```diff
@@ -189,28 +200,40 @@ func (c *celAdmissionController) Validate(
 
 	var allDecisions []PolicyDecisionWithMetadata = nil
 
-	addConfigError := func(err error, definition PolicyDefinition, binding PolicyBinding) {
-		wrappedError := fmt.Errorf("configuration error: %w", err)
-		switch p := definition.GetFailurePolicy(); p {
-		case Ignore:
-			klog.Info(wrappedError)
+	addConfigError := func(err error, definition *v1alpha1.ValidatingAdmissionPolicy, binding *v1alpha1.ValidatingAdmissionPolicyBinding) {
+		wrappedErrorForFail := fmt.Errorf("admission policy failed with error: %w", err)
+		wrappedErrorForIgnore := fmt.Errorf("Ignored ValidatingAdmissionPolicy %s failure for binding %s, due to FailurePolicy=Ignore. Error: %w", definition.Name, binding.Name, err)
+
+		policy := v1alpha1.Fail
+		if definition.Spec.FailurePolicy != nil {
+			policy = *definition.Spec.FailurePolicy
+		}
```

Comment:
Alternative to consider: remove this code and use `case v1alpha1.Fail, nil: // default to fail` below.